unfetter-discover/unfetter-ui
# Issue 1464

Requires `issue-1464` on `unfetter-ui`, `unfetter-store`, and `stix2pattern`

Associated PRs:
- https://github.com/unfetter-discover/stix2pattern/pull/38
- https://github.com/unfetter-discover/unfetter-store/pull/278

Instructions:
- Stop/remove containers, remove `unfetter-pattern-handler` image
- Restart stack
- AE => Add Analytic => Pseudocode => Stix Pattern, confirm `[process:pid = 3]` hits the api
- Switch Pseudocode to Sigma, and pick a rule at random and confirm it validates: `https://github.com/Neo23x0/sigma/tree/master/rules` -- Note, not all valid Sigma will translate
- Manually type a Sigma analytic, and confirm it validates/translations correctly - I had issues with getting proper YAML formatting.  Here is a simple Sigma that is easy to type: 

```yaml
detection:
  test:
    myvar: 3
  condition: test
```

- Create and edit analytics with text, stix pattern, and sigma Pseudocode, and confirm they behave / display as expected
- Go to AE => Filters, and confirm Stix pattern and sigma filters work as expected

fixes unfetter-discover/unfetter#1464
